server/app/views/questiontypes/MapQuestionFragment.html

```diff
@@ -5,17 +5,33 @@
   th:fragment="map(question, questionRendererParams)"
   th:with="mapQuestion=${question.createMapQuestion()},
              questionId=${'id-' + #strings.randomAlphanumeric(8)},
-             firstPathWithError=${mapQuestion.getFirstPathWithError()},
+              mapId=${'cf-map-' + #strings.randomAlphanumeric(8)},
+              firstPathWithError=${mapQuestion.getFirstPathWithError()},
              hasGroupErrors=${mapQuestion.getValidationErrors().get(question.getContextualizedPath())},"
   th:id="${questionId}"
   class="cf-question-map"
   th:classappend="${hasGroupErrors ? 'cf-question-field-with-error' : ''}"
   data-testid="questionRoot"
 >
   <!-- TODO(#11003): Build out applicant view of map question -->
-  This is a map question!
+  <div th:id="${mapId}" class="square-mobile"></div>
   <div
+    th:data-mapid="|${mapId}|"
+    class="hidden"
     th:if="${questionRendererParams.geoJson().isPresent()}"
-    th:text="${questionRendererParams.geoJson().get()}"
+    th:value="${questionRendererParams.geoJson().get()}"
   ></div>
```

Comment:
This div can be removed. It's no longer needed as all the data id and geojson are encoded in `window.app.data.maps`.